kernel/build-kernel.sh

```diff
@@ -0,0 +1,252 @@
+#!
+#!/bin/bash
```

Comment:
This needs to be on the first line, otherwise the kernel will use the default `/bin/sh` rather than `bash` I think.